Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -454,6 +456,8 @@
     - type: Lathe
       idleState: icon
       runningState: icon
+      unlitIdleState: unlit #DeltaV - added animation sprite layers
```

Comment:
add a space after the `#` for these: `unlit # DeltaV - added animation sprite layers`